ng-dev/utils/git/authenticated-git-client.ts

```diff
@@ -85,28 +85,35 @@ export class AuthenticatedGitClient extends GitClient {
     return {error};
   }
 
+  /** Gets an owned fork for the configured project of the authenticated user. */
+  async getForkOfAuthenticatedUser(): Promise<GithubRepo> {
+    const forks = await this.getForksForAuthenticatedUser();
+
+    if (forks.length === 0) {
+      throw Error('Unable to find fork a for currently authenticated user.');
+    }
+
+    return forks[0];
+  }
+
   /**
-   * Gets an owned fork for the configured project of the authenticated user, caching the determined
-   * fork repository as the authenticated user cannot change during action execution.
+   * Finds all forks owned by the currently authenticated user in the Git client,
+   *
+   * The determined fork repositories are cached as we assume that the authenticated
+   * user will not change during execution, or that no new forks are created.
    */
-  async getForkOfAuthenticatedUser(): Promise<GithubRepo> {
-    if (this._cachedForkRepo !== null) {
-      return this._cachedForkRepo;
+  async getForksForAuthenticatedUser(): Promise<GithubRepo[]> {
```

Comment:
should this be `getForksOfAuthenticatedUser` to match the other method naming convention?

Reply:
👍